conf/etc/consul_watcher_stop_io.json

```diff
@@ -0,0 +1,9 @@
+{
+    "watches": [
+          {
+            "type": "keyprefix",
+            "prefix": "cortx/base/ha/obj/node/",
+            "args": ["/etc/cortx/ha/ha_stop_io"]
```

Comment:
`/etc/cortx/`
is for configuration
we can use source path directly `/opt/seagate/cortx/ha/script/ha_stop_io`

Reply:
okay. @indrajitzagade  , changing the path to /opt/seagate/cortx/ha/script/ha_stop_io .  is that  fine ?

Reply:
I am fine
@ajaysrivas ?

Reply:
moved the file to /opt/seagate/cortx/ha/conf/script/ha_stop_io